README.md

```diff
@@ -111,6 +111,129 @@ kardinal flow ls
 
 Once you've deleted the flow, you can verify that the resources have been cleaned up by going to the dashboard again.
 
+## Develop with Kardinal + Tilt
+
+<details>
+<summary>Expand to see how to Develop with Kardinal + Tilt</summary>
+
+Is it possible to develop your app with Kardinal + [Tilt](https://tilt.dev/). Here’s how to do it::
+
+### Prerequisites:
+
+- [Kardinal CLI](https://github.com/kurtosis-tech/kardinal?tab=readme-ov-file#installation)
+- [Tilt](https://docs.tilt.dev/install.html)
+- Local K8s cluster, it could be [Minikube](https://minikube.sigs.k8s.io/docs/start) or [Docker desktop](https://docs.docker.com/desktop/kubernetes/) for instance
+- [Istio](https://istio.io/latest/docs/setup/install/istioctl/#install-istio-using-the-default-profile)
+
+### Deploy your application:
+
+#### Option 1 - Deploy your current Kardinal topology
```

Comment:
What do you think about naming Option 1: Use Tilt with existing Kardinal deployment and Option 2: Using Tilt without existing Kardinal deployment. At first it was a bit unclear the difference from the option headers but once reading I understood the difference

Reply:
great suggestion @tedim52, I'll do it